crates/services/sync/src/service/tests.rs

```diff
@@ -60,12 +61,15 @@ async fn test_new_service() {
         s.start_and_await().await.unwrap(),
         fuel_core_services::State::Started
     );
+    let mut last_value = 0;
     while let Some(h) = rx.recv().await {
+        last_value = h;
         if h == 16 {
             break
         }
     }
 
+    assert_eq!(last_value, 16);
```

Comment:
The loop only breaks on 16 so this is always true

Reply:
Previously the loop didn't work=) It was `None` because the import task failed at the start.

Reply:
ah I see